core/src/replay_stage.rs

```diff
@@ -546,6 +566,7 @@ impl ReplayStage {
             let mut duplicate_slots_tracker = DuplicateSlotsTracker::default();
             let mut duplicate_confirmed_slots: DuplicateConfirmedSlots =
                 DuplicateConfirmedSlots::default();
+            let mut repair_threshold_slots = RepairThresholdSlots::default();
```

Comment:
So how big could this table be? We probably need to give an estimate somewhere in comments.

Reply:
It's pruned every root.

Lower bound:
1 hash per slot, 32 slots since root:
`32B * 1 * 32 = 1KB`

Upper bound:
Currently each slot can have `100 - 33 = 67` versions reach the repair threshold. 
Based on the `wen_restart` numbers, we could support `65535` (7.3 hours) slots without a root. 
`32B * 67 * 65535 = 141 MB`.

lmk where and how much of this we should comment.

Reply:
Let's just put it above this line for now, when the variable is first defined.